packages/frontend/components/composed/journal/RelatedJournals/RelatedJournals.tsx

```diff
@@ -115,18 +124,21 @@ const fragment = graphql`
               storage
               ...CoverImageFragment
             }
-          }
-          ... on Item {
-            __typename
-            id
-            title
-            slug
-            published {
```

Comment:
Side note - Since this is related journals, I think we can safely say the target is a Collection (rather than spreading the same props to on Collection and on Item)

Reply:
Right— this PR should include that change.

Reply:
Oh! Thanks - I miss read the change 😅